src/modules/colorPicker/ColorPickerUI/Views/MainView.xaml

```diff
@@ -23,72 +25,71 @@
-                        <ColumnDefinition Width="36"/>
-                        <ColumnDefinition Width="Auto"/>
+                        <ColumnDefinition Width="36" />
+                        <ColumnDefinition Width="Auto" />
                     </Grid.ColumnDefinitions>
 
-                    <Border Background="{Binding ColorBrush}"
-                    HorizontalAlignment="Stretch"
-                    VerticalAlignment="Stretch"
-                    BorderBrush="{DynamicResource WindowBorderBrush}"
-                    BorderThickness="1"
-                    CornerRadius="4"/>
+                    <Border
+                        HorizontalAlignment="Stretch"
+                        VerticalAlignment="Stretch"
+                        Background="{Binding ColorBrush}"
+                        BorderBrush="{DynamicResource SurfaceStrokeColorDefaultBrush}"
+                        BorderThickness="1"
+                        CornerRadius="{DynamicResource ControlCornerRadius}" />
 
-                    <TextBlock 
-                       x:Name="ColorTextBlock"
-                       Margin="8,5,8,8"
-                       FontSize="16"
-                       FontWeight="SemiBold"
-                       Foreground="{DynamicResource PrimaryForegroundBrush}"
-                       Grid.Column="1"
-                       AutomationProperties.LiveSetting="Assertive"
-                       AutomationProperties.LabeledBy="{Binding ColorName}" 
-                       AutomationProperties.Name="{Binding ColorName}" 
-                       Text="{Binding ColorText}"/>
+                    <TextBlock
+                        x:Name="ColorTextBlock"
+                        Grid.Column="1"
+                        Margin="8,5,8,8"
+                        AutomationProperties.LiveSetting="Assertive"
+                        AutomationProperties.Name="{Binding ColorName}"
+                        FontSize="16"
+                        FontWeight="SemiBold"
+                        Text="{Binding ColorText}" />
                 </Grid>
 
-                <!--color format and color name - two lines-->
+                <!--  color format and color name - two lines  -->
                 <Grid Margin="2" Visibility="{Binding ShowColorName, Converter={StaticResource bool2VisibilityConverter}}">
                     <Grid.RowDefinitions>
-                        <RowDefinition Height="26"/>
-                        <RowDefinition Height="26"/>
+                        <RowDefinition Height="26" />
+                        <RowDefinition Height="26" />
                     </Grid.RowDefinitions>
                     <Grid.ColumnDefinitions>
-                        <ColumnDefinition Width="52"/>
-                        <ColumnDefinition Width="Auto"/>
+                        <ColumnDefinition Width="52" />
+                        <ColumnDefinition Width="Auto" />
                     </Grid.ColumnDefinitions>
```

Comment:
This might also be a single row grid, where the following 2 textblocks are in a (vertical) stackpanel. You could merge the two variants into one, and use a `Visibility=` for the second - optional - textblock.

Reply:
Please try this :)
[MainView.zip](https://github.com/microsoft/PowerToys/files/13114864/MainView.zip)

Reply:
`ColumnSpacing` is a UWP/WinUI property, unfortunately not available in WPF.

I'm leaving this out - as I think the current implementation was done because of the parent window (not) resizing. The PR is already pretty big, so I don't want to break more for now :).